code/modules/reagents/reagent_dispenser.dm

```diff
@@ -329,3 +335,27 @@
 		to_chat(user, "<span class='notice'>Sprayer refilled.</span>")
 		playsound(get_turf(src), 'sound/effects/refill.ogg', 50, 1, -6)
 		return 1
+		
+/obj/structure/reagent_dispensers/proc/wrenchAnchor(var/mob/user) //proc to wrench a wrenchable dispenser into place
```

Comment:
Did you copy this from machinery code? 

Might be a better idea to move the proc to the `/obj` level, and to remove some copy paste.